src/main/java/com/atharvadholakia/password_manager/data/Credential.java

```diff
@@ -0,0 +1,59 @@
+package com.atharvadholakia.password_manager.data;
+
+import jakarta.validation.constraints.NotBlank;
+import java.util.UUID;
+
+public class Credential {
+
+  private String id;
+
+  @NotBlank(message = "Servicename cannot be empty!")
+  private String servicename;
```

Comment:
Let's use `camelCase` for `servicename` i.e., change `servicename -> serviceName` everywhere in the project.